config/locales/en.yml

```
@@ -1988,7 +1988,7 @@ en:
           added to the OpenHistoricalMap database or this site, please send email to historic@openstreetmap.org with "TAKEDOWN REQUEST" in the subject line.
```

These requests will get a faster response if they go to the advisory group’s mailing list instead of the dormant public mailing list:

```suggestion
          added to the OpenHistoricalMap database or this site, please send email to ohm-admins@googlegroups.com with "TAKEDOWN REQUEST" in the subject line.
```

At some point, we may want to replace this e-mail address with a more official-looking one, but it shouldn’t really matter for the purpose of DMCA takedown notices.